### PR TITLE
Redesign top panel

### PR DESCRIPTION
**Pull request by erdkse**
This PR...

## Changes

-

## Fixes

-

## How to test it

-

## Screenshots

-

## Checklist

- [ ] tested locally
- [ ] added new dependencies
- [ ] updated the docs
- [ ] added a test

### CODE REVIEW

**Comment by erdkse** [comment]
We have 4 preview types. `Helm`, `Kustomization`, `Cluster`, `Helm configuration`. We may consider showing them in different color codes. Currently there are two color varients. For `Cluster` and the rest. Wdyt @julioramosest .


Screen Shot 2022-05-29 at 10 01 36

Screen Shot 2022-05-29 at 10 01 00

Screen Shot 2022-05-29 at 10 02 01

Screen Shot 2022-05-29 at 10 02 23

File: src/components/organisms/PageHeader/ClusterSelection.styled.tsx [hunk 1]
```diff
@@ -1,42 +1,101 @@
...
+  if (previewType === 'kustomization') {
+    color = rgbaRatio ? rgba(Colors.cyan, rgbaRatio) : Colors.cyan;
+  }
 
-  &:hover {
-    color: ${Colors.blue6};
+  return isBoolean(isInPreviewMode) ? (isInPreviewMode ? color : fallBackColor) : color;
+};
+
+export const Button = styled(({children, previewType, isInPreviewMode, ...rest}: RawButtonProps) => (
```
Comment (WitoDelnat) [inline src/components/organisms/PageHeader/ClusterSelection.styled.tsx:51]: Unused properties should be removed?

File: src/components/organisms/PageHeader/ClusterSelection.tsx [hunk 2]
```diff
@@ -150,61 +134,117 @@ const ClusterSelection = ({previewResource}: {previewResource?: K8sResource}) =>
...
-            </>
-          )}
-        </S.ClusterStatus>
+            )}
+          </S.ClusterStatus>
+        </>
       )}
+      <>
+        {isKubeConfigPathValid && activeProject && (
+          // <Tooltip mouseEnterDelay={TOOLTIP_DELAY} mouseLeaveDelay={0} title={ClusterModeTooltip} placement="right">
```
Comment (WitoDelnat) [inline src/components/organisms/PageHeader/ClusterSelection.tsx:225]: commented code

**Review by WitoDelnat — APPROVED** [review]
Three nitpicks but I've already approved. Great work!

**Comment by WitoDelnat** [comment]
Bit a nitpick but can this menu get a bit more room to breath? It's rather small and jammed against the right wall.

Screenshot 2022-05-30 at 09 16 29

File: src/components/organisms/PageHeader/ClusterSelection.tsx (hunk 2 above)
Comment (julioramosest) [reply src/components/organisms/PageHeader/ClusterSelection.tsx:225]: @erdkse good work! I agree with Wito that the dropdown could benefit from some padding on the right for the longest item. Same for the "+" dropdown, you can add some padding on the right when displayed, so "new form local folder" has a bit more air to the right.

File: src/components/organisms/PageHeader/ClusterSelection.styled.tsx (hunk 1 above)
Comment (erdkse) [reply src/components/organisms/PageHeader/ClusterSelection.styled.tsx:51]: they cause an error if I don't extract them. I couldn't find a way. Do you have an idea @WitoDelnat ?

File: src/components/organisms/PageHeader/ClusterSelection.tsx (hunk 2 above)
Comment (erdkse) [reply src/components/organisms/PageHeader/ClusterSelection.tsx]: Ok thank you folks for the reviews, I made fixes according to them. Now I am merging and I can add further wishes on following PRs

**Pull request merged**